lib/core/client.js

```diff
@@ -929,6 +932,41 @@ function _resume (client, sync) {
   }
 }
 
+async function writeIterable (socket, contentLength, header, body) {
+  try {
+    if (contentLength === null) {
+      socket.write(`${header}transfer-encoding: chunked\r\n`, 'ascii')
+    } else {
+      socket.write(`${header}content-length: ${contentLength}\r\n\r\n`, 'ascii')
+    }
+    for await (const chunk of body) {
+      // if (typeof chunk !== 'string' && !util.sBuffer(chunk)) {
+      //   throw new Error('Body iterable must yield either string or buffer')
+      // }
```

Comment:
Uncommented?

Reply:
not sure why i commented this; will fix.

Reply:
fwiw you asked this to be removed previously so its gone now!